js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -38,22 +48,27 @@ export default function AssetGroup() {
 			/>
 			<AssetGroupSection />
 			<StepContentFooter>
-				<AppButton
-					isTertiary
-					data-action={ ACTION_SKIP_ASSETS }
-					disabled={
-						// TODO: Change to only validate campaign data.
-						! isValidForm ||
-						isSubmitted ||
-						currentAction === ACTION_COMPLETE
-					}
-					loading={
-						isSubmitting && currentAction === ACTION_SKIP_ASSETS
-					}
-					onClick={ handleSubmit }
-				>
-					{ __( 'Skip adding assets', 'google-listings-and-ads' ) }
-				</AppButton>
+				{ ( isCreation || isEmptyAssetGroup ) && (
```

Comment:
Correct me if I'm wrong, but isn't true that in Creation mode the assets will be always empty?

Reply:
Currently, all campaigns created by GLA always have at least an associated asset group. In GLA, we want to prompt users that the assets could be added more to improve their ads' performance, so there is a logic specific to GLA to identify if an existing asset group is "empty". Ref: pcTzPl-1c9-p2#comment-2001

Therefore, when creating, the asset group is "non-existent", while when editing, the asset group already exists but it needs to distinguish if it's empty. However, the part about determining an empty asset group hasn't been implemented yet, so it's not yet certain whether the asset group is also considered "empty" when creating a new campaign.